File: test/golden/request/ruby/0.rb
# Built by LucyBot. www.lucybot.com
uri = URI("http://api.lucybot.com/request/languages")
http = Net::HTTP.new(uri.host, uri.port)
request = Net::HTTP::Get.new(uri.request_uri)
@result = http.request(request).body
puts @result

File: test/golden/request/javascript/0.js
// Built by LucyBot. www.lucybot.com
var url = "http://api.lucybot.com/request/languages";
$.ajax({
  url: url,
  method: 'GET',
}).done(function(result) {
  console.log(result);
}).fail(function(err) {
  throw err;
});
